src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -325,6 +325,12 @@ public Timestamp preaccept(TxnId txnId, Keys keys)
         return time.uniqueNow(max);
     }
 
+    @Override
+    public Timestamp uniqueNow(TxnId greaterThan)
+    {
+        return time.uniqueNow(greaterThan);
```

Comment:
If node time service already factors this out why does it need to be in both the command store implementations?

Reply:
Fair. I'll expose the time service instead.